.github/workflows/pull-request.yml

```diff
@@ -13,6 +13,8 @@ jobs:
   build:
     name: 'Build & canary release'
     runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
Don't forget to add this option into other build related workflows(release, measure-size)

Reply:
Done